modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APISortRelevantObjectEntryModelListenerTest.java

```diff
@@ -326,8 +327,8 @@ private JSONObject _addObjectEntry() throws Exception {
 	private static final String _OBJECT_FIELD_NAME =
```

Comment:
Is possible using the same method here?

Reply:
I don't think so because that method is meant to generate object definition names. We should use a separate method for object field names or generalize the existing method as something like `randomFirstLetterString`... 

In any case I would do that in a separate ticket.